.github/release_template.md

```diff
@@ -0,0 +1,33 @@
+<!--
+This is the template to release a new version on the App Store
+-->
+
+Release a new version
+## TASKS:
+- [ ] [GIT] Create branch release_X.X.X (freeze the code)
+- [ ] [DEV] Update version number X.X.X
+- [ ] [DIS] Update changelogs:
+     - [ ] [DEV] Update changelog https://github.com/owncloud/ios-app/blob/master/CHANGELOG.md
+     - [ ] [GIT] Add the new release on GitHub https://github.com/owncloud/ios-app/releases
+- [ ] [QA] Design Test plan
+- [ ] [QA] Regression Test plan
+- [ ] [DIS] Update screenshots if needed
+- [ ] [DIS] Release to appstore
+- [ ] [DOC] Update user manual with the new functionalities
+- [ ] [GIT] Merge branch release_X.X.X in master and develop
+- [ ] [GIT] Create tag and sign it "version_X.X.X"
+
+If it is required to update third party
+- [ ] [DIS] Update THIRD_PARTY.txt
+
+If it is required to update the iOS-SDK version:
+- [ ] [GIT] Create branch library release_Y.Y.Y (freeze the code) 
```

Comment:
We're following the vanilla OneFlow concept (http://endoflineblog.com/oneflow-a-git-branching-model-and-workflow#starting-a-release-branch), which names that branch `release/[major].[minor].[patch]`.